workloads/network-perf/common.sh

```diff
@@ -43,29 +43,39 @@ export_defaults() {
   export pin=true
   export networkpolicy=${NETWORK_POLICY:=false}
   export multi_az=${MULTI_AZ:=true}
+  export baremetalCheck=$(oc get infrastructure cluster -o json | jq .spec.platformSpec.type)
   zones=($(oc get nodes -l node-role.kubernetes.io/workload!=,node-role.kubernetes.io/worker -o go-template='{{ range .items }}{{ index .metadata.labels "topology.kubernetes.io/zone" }}{{ "\n" }}{{ end }}' | uniq))
 
-  # If multi_az we use one node from the two first AZs
-  if [[ ${multi_az} == "true" ]]; then
-    # Get AZs from worker nodes
-    log "Colocating uperf pods in different AZs"
-    if [[ ${#zones[@]} -gt 1 ]]; then
-      export server=$(oc get node -l node-role.kubernetes.io/worker,node-role.kubernetes.io/workload!="",node-role.kubernetes.io/infra!="",topology.kubernetes.io/zone=${zones[0]} -o jsonpath='{range .items[*]}{ .metadata.labels.kubernetes\.io/hostname}{"\n"}{end}' | head -n1)
-      export client=$(oc get node -l node-role.kubernetes.io/worker,node-role.kubernetes.io/workload!="",node-role.kubernetes.io/infra!="",topology.kubernetes.io/zone=${zones[1]} -o jsonpath='{range .items[*]}{ .metadata.labels.kubernetes\.io/hostname}{"\n"}{end}' | tail -n1)
-    else
-      log "At least 2 worker nodes placed in different topology zones are required"
-      exit 1
-    fi
-  # If multi_az is disabled we use the two first nodes from the first AZ
+  #If using baremetal we use different query to find worker nodes
+  if [[ "${baremetalCheck}" == '"BareMetal"' ]]; then
+  log "Colocating uperf pods for baremetal"
```

Comment:
Bash doesn't really require indentation sure, but mind indenting these lines inside the if block for constancy with other blocks in the code?

Reply:
spaces included now